jetty9/src/main/docker/jetty_start.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+DIR=$(dirname $0)
+JETTY_START_COMMAND="/scripts/jetty/jetty-start-command"
+
+if [ ! -f $JETTY_START_COMMAND ]; then
+  source $DIR/gen_start_command.sh $JETTY_START_COMMAND
```

Comment:
I think addressing container start time like this might be a premature optimization, you should gather some real world metrics related to this in this specific usage. For any application of beyond trivial size the calculation of this command line is likely to be a very minor portion of overall startup time, at least for anything I have seen thus far.  It would be unfortunate to add an option like this, document it and have users enable it thinking there would be a dramatic change and not have it materialize.